libcxx/include/stdatomic.h

```diff
@@ -154,10 +154,14 @@ using std::atomic_long _LIBCPP_USING_IF_EXISTS;
 using std::atomic_ulong _LIBCPP_USING_IF_EXISTS;
 using std::atomic_llong _LIBCPP_USING_IF_EXISTS;
 using std::atomic_ullong _LIBCPP_USING_IF_EXISTS;
+#  ifndef _LIBCPP_HAS_NO_CHAR8_T
```

Comment:
The using-if-exists attribute should handle this, is there a reason why you're adding this `#if`?

Reply:
It looks like GCC doesn't have the `__using_if_exists__` attribute, so the `generic-gcc-cxx11` test failed until I added the `#if`.

Reply:
Right. `char8_t` is not supported before C++20 so this starts failing when we don't have the attribute. Ok, I think that's acceptable.